tests/httpMethodsTest.test.js

```diff
@@ -0,0 +1,71 @@
+const request = require("supertest");
+const fs = require("fs");
+const { TestScheduler } = require("jest");
+const { items } = require("../dataBase");
+const app = require("../app");
+const DataBase = require("../dataBase");
+const { count } = require("console");
+
+describe("POST route", ()=> {
+
+    it("should post a new short id successfully", async () => {
+        const response = await request(app)
+        .post("/api/shorturl/new")
+        .type("form")
+        .send({url : "https://www.youtube.com/?hl=iw&gl=IL"});
+        expect(response.status).toBe(200);
+    });
```

Comment:
Next time check the response is fine, and if you really want to be sure you can check the DB for a new entry